### PR TITLE
LPS-201821 Ensures the creator user exists when copying a layout

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPS-201821

cc: @markomihalovics

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/helper/LayoutCopyHelperImpl.java [hunk 1]
```diff
@@ -713,6 +726,34 @@ private String _getTypeSettings(Layout sourceLayout, Layout targetLayout) {
 		).buildString();
 	}
 
+	private User _getUser(
+			Layout sourceLayout, Layout targetLayout,
+			ServiceContext currentServiceContext)
```
Comment (ealonso) [inline modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/helper/LayoutCopyHelperImpl.java:731]: @lfbesada here the name should be serviceContext

Comment (lfbesada) [reply modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/helper/LayoutCopyHelperImpl.java:731]: Thanks for catching it! :heart:

**Comment by lfbesada** [comment]
@ealonso Thanks for the feedback :heart:  I have changed my first commit to add it. Regards

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**